main.py

```
@@ -481,21 +483,50 @@ async def _handle_graph_update(bot: TGraphBot, channel: discord.TextChannel) ->
+            current_update.isoformat() if current_update else "None",
+            bot.update_tracker.next_update.isoformat() if bot.update_tracker.next_update else "None"
+        )
+
+        # 3. Create a temporary tracker state for embed generation
+        temp_tracker_state = bot.update_tracker.get_state()
+
+        try:
+            # 4. Delete old messages
+            await bot.graph_manager.delete_old_messages(channel)
+            logging.debug("Successfully deleted old messages")
+        except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)